queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT * FROM animals WHERE date_of_birth BETWEEN date '01-01-2016' AND date '01-01-2019';
```

Comment:
- [ ] Kindly make sure that the end date is `'2019/31/12' `as per the requirements.
- [ ] Also make sure to display the screenshot for this as the previous one was not right.

Reply:
Hi [sja-thedude](https://github.com/sja-thedude),
I'm a bit confused about your requirements as I did not see the requirement for the date format. Kindly direct me to it. Thanks.